packages/calling-stateful-client/src/LocalVideoStreamVideoEffectsSubscriber.ts

```diff
@@ -48,17 +48,17 @@ export class LocalVideoStreamVideoEffectsSubscriber {
     this._localVideoStreamEffectsAPI.off('effectsError', this.effectsError);
   };
 
-  private effectsStarted = (effectName: VideoEffectName): void => {
+  private effectsStarted = (effects: VideoEffectName[]): void => {
     this.updateEffectsState({
       isActive: true,
-      effectName: effectName
+      effectName: effects[0]
```

Comment:
As part of background effects we are updating the stateful to also provide an array of effects, this will be a separate PR